src/panels/lovelace/components/hui-conditional-base.ts

```diff
@@ -17,6 +17,8 @@ export class HuiConditionalBase extends ReactiveElement {
 
   @property() protected _config?: ConditionalCardConfig | ConditionalRowConfig;
 
+  @property({ type: Boolean, reflect: true }) public invisible = false;
```

Comment:
Might be better name?
```suggestion
  @property({ type: Boolean, reflect: true }) public hidden = false;
```

Reply:
I agree.